### DISCUSSION TITLE
VScode预览pdf无法显示。

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#109] [Q&A]
你好，
今天在使用VScode，发现pdf无法显示，但是可以在文件夹里找到然后用Acrobat打开是正常的。这会是哪里出问题了？
![Screenshot 2022-04-10 212744](https://user-images.githubusercontent.com/45897471/162620547-b340e9e5-f7fc-4dce-b805-511664da5476.jpg)

**Comment by tanukihee** [comment]
什么原因都有可能，但不可能是 whu-thesis 的问题

**Reply by earthmoon** [reply]
> 什么原因都有可能，但不可能是 whu-thesis 的问题

多次重启vscode好了。
